src/redux/table/table-actions.js

```diff
@@ -0,0 +1,9 @@
+import { createAction } from '@reduxjs/toolkit';
+
+const fetchTransactions = createAction('transactions/fetch');
+
+const getBalance = createAction('balance/getBalance');
+
+const actions = { fetchTransactions, getBalance };
+
+export default actions;
```

Comment:
Please rename this folder and files inside to "transactionsTable" or something like this

Reply:
renamed